src/api/__specs__/charities-spec.js

```diff
@@ -13,6 +13,7 @@ describe('charities', () => {
   };
 
   beforeEach(() => {
+    callback = sinon.spy();
```

Comment:
I saw that just after I submitted my pr. I'll fix it in the morning.
It's also a syntax thing fort groking const. Even though I know const refers to the pointer we are making changes to the class state so highlighting it as let will hopefully let people know that this object is changing